plugins/GSdx/GSRendererOGL.cpp

```diff
@@ -1344,12 +1344,16 @@ void GSRendererOGL::DrawPrims(GSTexture* rt, GSTexture* ds, GSTextureCache::Sour
 
 			// We need the palette to convert the depth to the correct alpha value.
 			if (!tex->m_palette) {
```

Comment:
Well, is the code is executed also with 32 bit textures? I didn't really inspected well the code, but to me seemed so. It may be needed a bit of refactoring in any case.

Reply:
On GPU it is a depth texture (so float number) but on GS side it is a PSM_PSMT8H (i.e. 8 bits palettes + 24 bits padding).

I don't remember why I did this ! Maybe we could just always load the clut from the texture cache (even when format is special). It should be mostly free now. I really should post my gsdump somewhere.